Jint/Runtime/Interop/MethodInfoFunctionInstance.cs

```diff
@@ -116,7 +118,18 @@ public JsValue Invoke(MethodInfo[] methodInfos, JsValue thisObject, JsValue[] js
                 }
             }
 
-            throw new JavaScriptException(Engine.TypeError, "No public methods with the specified arguments were found.");
+            var errStr = $"{_methodName}(";
+            if (arguments.Length > 0)
+            {
+                for (var i = 0; i < arguments.Length; i++)
+                {
+                    errStr += $"{arguments[i].Type}, ";
```

Comment:
is this safe if `null` or `undefined` are passed as arguments?

Reply:
Yep! Those aren't the raw arguments, they're `JsValue` instances, so the arg and `Type` will exist, but with a value of `Null` or `Undefined`

Reply:
👍